index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700&display=swap" rel="stylesheet">
+    <title>Portfolio Website</title>
+</head>
+<body>
+    <!-- HEADER SECTION-->
+    <header class="header__section">
+        <nav class="navbar__section">
+            <h2 class="navbar__logo">Logo</h2>
```

Comment:
Kindly I suggest you make sure the logo has a link `a` tag to make your project more professional and easy to use. 😄